gradle/util.gradle

```diff
@@ -1,27 +0,0 @@
-tasks.register("artifacts") {
```

Comment:
I suspect no one is using this task but if so speak up and I can add it somewhere